src/main/java/GameField.java

```diff
@@ -0,0 +1,78 @@
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class GameField {
+
+    private Rectangle field;
+
+    private Collection<GameObject> gameObjects;
+
+    private Rectangle movedBody;
+
+    public GameField(Point size){
```

Comment:
please use least possible modifier. same everywhere

Reply:
Tests used all classes, if I'll change modifier on private, they won't be able use them

Reply:
I didnt mean private but least **possible**. So in our case its private-package

Reply:
fixed, except Game class, it's expected to be used outside of the package in the future

Reply:
so make it public only when its necessary